daml-assistant/daml-helper/src/DA/Daml/Helper/Ledger.hs

```diff
@@ -48,6 +52,7 @@ import System.FilePath
 import System.IO.Extra
 import System.Process.Typed
 
+import Com.Daml.Ledger.Api.V1.TransactionFilter --TODO: HL mirror
```

Comment:
What's that todo about?

Reply:
that's copy pasta, no idea what the todo is about. I'll remove it from here.